lookout/style/typos/general.py

```diff
@@ -0,0 +1,83 @@
+from typing import Any, Mapping, Optional
+
+from google_drive_downloader import GoogleDriveDownloader as gdd
+import pandas
+
+from lookout.style.common import merge_dicts
+from lookout.style.typos.preprocessing import filter_splits, print_frequencies
+from lookout.style.typos.utils import Columns, flatten_df_by_column
+
+
+DRIVE_DATASET_ID = "1muNVWPe68XK8SFvqIv3V728NmkT46aTx"
+DRIVE_FASTTEXT_ID = "1hCOIwKn-QZLVv1S385HxyNMeERgKGIvo"
```

Comment:
Fasttest vectors must be trained from scratch (they are not generic, are they?), we may not download it.

Reply:
Ok, then I will add that training to this function (it is also done on the raw data). Is it better to do it in this PR, or the next one?

Reply:
Better in a separate one

Reply:
@vmarkovtsev Btw, I use pretrained vectors from [here](https://github.com/facebookresearch/fastText/blob/master/docs/pretrained-vectors.md), so I should download them every time to train new fasttext model? It's quite heavy (6Gb). We definitely should provide this functionaly, but I don't think it's good to do it every time in `prepare_data`, it's better to use our pretrained ones in most cases.